sources/models/src/aws-k8s-1.15/override-defaults.toml

```diff
@@ -5,7 +5,7 @@ template-path = "/usr/share/templates/containerd-config-toml_aws-k8s"
 # Kubernetes.
 
 [services.kubernetes]
-configuration-files = ["kubelet-env", "kubelet-config", "kubelet-kubeconfig", "kubernetes-ca-crt"]
+configuration-files = ["kubelet-env", "kubelet-config", "kubelet-kubeconfig", "kubernetes-ca-crt", "proxy-env"]
```

Comment:
Does the configuration file need to be added to aws-dev?

Reply:
I removed `"proxy_env"` from all `services.` metadata and created a separate service metadata table to house it